frontend/src/components/contextMenu/ContextMenuModel.ts

```
@@ -0,0 +1,65 @@
+  }
+
+  get htmlId(): string {
+    return `context-menu-popover-${this.id}`;
+  }
+
+  get anchorVar(): string {
+    return `--${this.htmlId}`;
+  }
+
+  executeAction(action: IContextMenuAction): void {
+    action.onAction({ menuRect: this.menuRect! });
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)